api/occupancy/OccupancyOperations.php

```diff
@@ -7,18 +7,35 @@
    * @author Stan Callewaert
    */
 
+use Dotenv\Dotenv;
+use MongoDB\Collection;
+
 class OccupancyOperations
 {
     const UNKNOWN = 'https://api.irail.be/terms/unknown';
     const LOW = 'https://api.irail.be/terms/low';
     const MEDIUM = 'https://api.irail.be/terms/medium';
     const HIGH = 'https://api.irail.be/terms/high';
 
+    public static function getOccupancy($vehicle, $date)
+    {
+        $dotenv = new Dotenv(dirname(dirname(__DIR__)));
+        $dotenv->load();
+        $mongodb_url = getenv('MONGODB_URL');
+        $mongodb_db = getenv('MONGODB_DB');
+
+        $m = new MongoDB\Driver\Manager($mongodb_url);
+        $occupancy = new MongoDB\Collection($m, $mongodb_db, 'occupancy');
+
+        // If we ever start using a date as parameter the parameter should be put here as date
+        return $occupancy->find(array('vehicle' => $vehicle, 'date' => $date));
+    }
+
     public static function NumberToURI($occupancy)
     {
-        if ($occupancy < 1/3) {
+        if ($occupancy < 2/3) {
             return self::LOW;
-        } elseif ($occupancy <= 2/3) {
+        } elseif ($occupancy <= 4/3) {
```

Comment:
4/3? If i'm right, the occupancy score can be 0, 1 or 2?

Can you document before this function what kind of number `occupancy` should be (between what intervals)? This is very important for future use in each case...